### ISSUE TITLE
Error when using Real Weather Module

### ISSUE THREAD

**Issue by alpha117** [ALiVEOS/ALiVE.OS#450]
![image](https://user-images.githubusercontent.com/30005040/32057782-29765bf6-ba60-11e7-8091-908d3b7b7ff6.png)

I selected ENGLAND/LONDON and set the other options to real weather

**Comment by friznit** [comment]
Must be because air pollution coefficient is outside of game tolerances again.....

Try "England/London"  It might be case sensitive?

**Comment by alpha117** [comment]
OK, let you know
Thanks for the quick reply

**Comment by alpha117** [comment]
No Luck same error

**Comment by alpha117** [comment]
France/Paris same error

**Comment by tupolov** [comment]
CITY/COUNTRY

**Comment by alpha117** [comment]
OK, let you know

Thanks

**Comment by alpha117** [comment]
Nope still errors

**Comment by tupolov** [comment]
Post your RPT

**Comment by alpha117** [comment]
Morning,
I would if I knew where  the RPT is.

As you might have gathered I'm new to all this :-)

**Comment by bradon** [comment]
fnc_getRealWeather.sqf line 120
Maybe he's getting no/invalid response that isn't being handled?

**Comment by alpha117** [comment]
Good point, has anyone got a valid pair that works for them. I can then prove if that is the issue

**Comment by alpha117** [comment]
cool, I'll what for the 'fix'


Thanks

**Comment by trapw0w** [comment]
Can't replicate. Presume this has been resolved?
